Sources/Containerization/ContainerManager.swift

```diff
@@ -0,0 +1,375 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the Containerization project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+#if os(macOS)
+
+import ContainerizationError
+import ContainerizationOCI
+import ContainerizationOS
+import Foundation
+import SendableProperty
+import ContainerizationExtras
+import Virtualization
+import vmnet
+
+/// A type that manages containers and their resources within a root directory.
+public struct ContainerManager: Sendable {
```

Comment:
Is the new guidance going to be for mostly everyone to use this type going forward? I'm debating whether it makes sense for the network protocol and types to be outside of this type.